ksql-common/src/main/java/io/confluent/ksql/util/timestamp/StringTimestampExtractionPolicy.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.ksql.util.timestamp;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.apache.kafka.streams.StreamsConfig;
+
+import java.util.Map;
+
+import io.confluent.ksql.util.KsqlConfig;
+
+public class StringTimestampExtractionPolicy implements TimestampExtractionPolicy {
```

Comment:
Java docs? / final class?

Docs could explain about the TIMESTAMP_FORMAT.

Reply:
yep - i need to update the docs. Will do that in a follow up if this can get in before code freeze.